pandas/tseries/tests/test_period.py

```diff
@@ -2537,6 +2540,16 @@ def test_searchsorted(self):
             ValueError, 'Different period frequency: H',
             lambda: pidx.searchsorted(pd.Period('2014-01-01', freq='H')))
 
+    def test_round_trip_pickle(self):
+
+        p = Period('2000Q1')
+        pickle_path = os.path.join(tm.get_data_path(),
+                                   'period.pickle')
+
+        with open(pickle_path, 'wb') as f: pickle.dump(p, f)
```

Comment:
do you mean in pandas/io/tests/generate_legacy_pickles?

On Wed, Aug 5, 2015 at 6:00 PM, Jeff Reback <notifications@github.com>
wrote:

> In pandas/tseries/tests/test_period.py
> <https://github.com/pydata/pandas/pull/10441#discussion_r36359951>:
>
> > @@ -2537,6 +2540,16 @@ def test_searchsorted(self):
> >              ValueError, 'Different period frequency: H',
> >              lambda: pidx.searchsorted(pd.Period('2014-01-01', freq='H')))
> >
> > +    def test_round_trip_pickle(self):
> > +
> > +        p = Period('2000Q1')
> > +        pickle_path = os.path.join(tm.get_data_path(),
> > +                                   'period.pickle')
> > +
> > +        with open(pickle_path, 'wb') as f: pickle.dump(p, f)
>
> also need to add a period in pandas/io/tests/generate_legacy_data.py
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/10441/files#r36359951>.
>